doc/api/date/date-to-parts-formatter.md

```diff
@@ -0,0 +1,190 @@
+
+- `second`
+
+  The string used for the second, e.g., `"07"` or `"42"`.
+
+- `timeZoneName`
+
+  The string used for the name of the time zone, e.g., `"EST".`
+
+- `weekday`
+
+  The string used for the weekday, e.g., `"M"`, `"Monday"`, `"Montag".`
+
+- `year`
+
+  The string used for the year, e.g., `"2012"`, `"96".`
+
+
+### Example
+
+Prior to using any date methods, you must load
+`cldr/main/{locale}/ca-gregorian.json`, `cldr/main/{locale}/timeZoneNames.json`,
+`cldr/supplemental/timeData.json`, `cldr/supplemental/weekData.json`, and the
+CLDR content required by the number module. Read [CLDR content][] if you need
+more information.
+
+[CLDR content]: ../../../README.md#2-cldr-content
+
+You can use the static method `Globalize.dateToPartsFormatter()`, which uses the
+default locale.
+
+```javascript
+var formatter;
+
+Globalize.locale( "en" );
+formatter = Globalize.dateToPartsFormatter();
+
+formatter( new Date( 2010, 10, 30, 17, 55 ) );
+// > [
+//   { "type": "month", "value": "11" },
+//   { "type": "literal", "value": "/" },
+//   { "type": "day", "value": "30" },
+//   { "type": "literal", "value": "/" },
+//   { "type": "year", "value": "2010" }
+// ]
+```
+
+You can use the instance method `.dateToPartsFormatter()`, which uses the instance locale.
+
+```javascript
+var enFormatter = Globalize( "en" ).dateToPartsFormatter(),
+  deFormatter = Globalize( "de" ).dateToPartsFormatter();
+
+enFormatter( new Date( 2010, 10, 30, 17, 55 ) );
+// > [
+//   { "type": "month", "value": "11" },
+//   { "type": "literal", "value": "/" },
+//   { "type": "day", "value": "30" },
+//   { "type": "literal", "value": "/" },
+//   { "type": "year", "value": "2010" }
+// ]
+
+deFormatter( new Date( 2010, 10, 30, 17, 55 ) );
+// > [
+//   { type: 'day', value: '30' },
+//   { type: 'literal', value: '.' },
+//   { type: 'month', value: '11' },
+//   { type: 'literal', value: '.' },
+//   { type: 'year', value: '2010' }
+// ]
+```
+
+The information is available separately and it can be formatted and concatenated
+again in a customized way. For example by using [`Array.prototype.map()`][],
+[arrow functions][], a [switch statement][], [template literals][], and
+[`Array.prototype.reduce()`][].
+
+[`Array.prototype.map()`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map
+[arrow functions]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions
+[switch statement]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch
+[template literals]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
+[`Array.prototype.reduce()`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce
+
+```javascript
+var formatter;
+
+Globalize.locale( "en" );
+formatter = Globalize.dateToPartsFormatter();
+
+formatter( new Date( 2010, 10, 30, 17, 55 ) ).map(({type, value}) => {
```

Comment:
> since a `join("")` wouldn't be enough

Why? Check the parts output of a date and time case https://github.com/globalizejs/globalize/pull/700/files#diff-5e625c00d307f2b886e1b33225953053R165

Having said that, I think it's valid including a date and time example.